Formula/glibc.rb

```diff
@@ -195,4 +204,15 @@ def post_install
     safe_system "#{lib}/libc.so.6", "--version"
     safe_system "#{bin}/locale", "--version"
   end
+
+  def ld_so_conf
+    <<~EOS
+      include #{prefix}/etc/ld.so.conf.d/*.conf
```

Comment:
This directory does not exist in the current bottle. Is it being created somehow? Why, and with what contents?